README.rst

```diff
@@ -95,8 +93,55 @@ Define a Configuration Model
                 User: ["password"]
             }
 
+
+Field Defaults
+^^^^^^^^^^^^^^
+
+For regular Pydantic fields, you must explicitly configure defaults using  `Field
+(default=...)` or `Field(default_factory=lambda: ...)` as specified in  the  `Pydantic
+documentation <https://docs.pydantic.dev/2.10/concepts/fields/#default-values>`_.
+
+**NOTE:** Marking a field as ``Optional`` or using ``... | None`` does *not* automatically set the field's default to `None`. You must set this explicitly if you want the field to be optional:
+
+.. code-block:: python
+
+    from pydantic import Field
+
+    class ConfigModel(ConfigurationModel):
+        optional_field: int | None = DjangoModelRef(SomeModel, "some_field", default=None)
+
+For ``DjangoModelRef``, the default value handling follows these rules:
+
+You can provide explicit defaults using the ``default`` or ``default_factory`` kwargs,
+similar to regular Pydantic fields:
+
+.. code-block:: python
+
+    class ConfigModel(ConfigurationModel):
+        # Explicit string default
+        field_with_explicit_default = DjangoModelRef(SomeModel, "some_field", default="foobar")
+        
+        # Explicit default factory for a list
+        field_with_explicit_default_factory: list[str] = DjangoModelRef(
+            SomeModel, "some_other_field", default_factory=list
+        )
+
+When no explicit default is provided, the default is derived from the referenced Django field:
+
+1. If the Django field has an explicit default, that default will be used.
+
+2. If no explicit default is set but the field has ``null=True`` set:
+    - The default will be set to ``None``
+    - The field will be optional
+
+3. If no explicit default is provided and the field is not nullable, but has 
+   ``blank=True`` **and** it is a string-type field:
+    - The default will be an empty string
+    - The field will be optional
```

Comment:
Minor thing, but the formatting of these is different when rendered

Reply:
Also I assume otherwise, if optionality, the key will not be in the output data?